packages/perseus-editor/src/components/interactive-graph-settings.tsx

```diff
@@ -545,112 +559,139 @@ class InteractiveGraphSettings extends React.Component<Props, State> {
-                                            },
-                                        )}
-                                    </select>
-                                </label>
-                            </div>
-                        </div>
+                        <View style={styles.spaceTop}>
+                            <LabeledRow
+                                label="Ruler label:"
+                                style={styles.resetSpaceTop}
+                            >
+                                <SingleSelect
+                                    id="ruler-label-select"
+                                    selectedValue={this.props.rulerLabel}
+                                    onChange={(newValue) => {
+                                        this.change({rulerLabel: newValue});
+                                    }}
+                                    placeholder="None"
+                                    style={styles.singleSelectShort}
+                                >
+                                    <OptionItem
+                                        value=""
+                                        label="None"
+                                        horizontalRule="full-width"
+                                    />
+                                    <OptionItem value="mm" label="Milimeters" />
+                                    <OptionItem
+                                        value="cm"
+                                        label="Centimeters"
+                                    />
+                                    <OptionItem value="m" label="Meters" />
+                                    <OptionItem
+                                        value="km"
+                                        label="Kilometers"
+                                        horizontalRule="full-width"
+                                    />
+                                    <OptionItem value="in" label="Inches" />
+                                    <OptionItem value="ft" label="Feet" />
+                                    <OptionItem value="yd" label="Yards" />
+                                    <OptionItem value="mi" label="Miles" />
+                                </SingleSelect>
+                            </LabeledRow>
+                            <LabeledRow label="Ruler ticks:">
+                                <SingleSelect
+                                    id="ruler-ticks-select"
+                                    selectedValue={`${this.props.rulerTicks}`}
+                                    onChange={(newValue) => {
+                                        this.change({rulerTicks: newValue});
+                                    }}
+                                    placeholder="10"
+                                    style={styles.singleSelectShort}
+                                >
+                                    {RULER_TICKS.map((value) => (
```

Comment:
If this is the only usage of `RULER_TICKS` I'd be inclined not to create a const as it adds some indirection to understand what it is.